client/src/main/java/com/paypal/selion/platform/mobile/android/UiTextView.java

```diff
@@ -15,10 +15,26 @@
 
 package com.paypal.selion.platform.mobile.android;
 
-public class UiTextView extends UiObject {
+import com.paypal.selion.platform.mobile.elements.MobileTextField;
+
+public class UiTextView extends UiObject implements MobileTextField {
 
     public UiTextView(String locator) {
         super(locator);
     }
 
+    @Override
+    public void setValue(CharSequence... keysToSend) {
+        String text;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using Literals in Conditional Statements](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2360069776/issues/source?bid=2928425&fileBranchId=3219780#l29)